common-npm-packages/azure-arm-rest-v2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -152,13 +152,12 @@
   "loc.messages.MethodNotImplementedError": "Method '%s' is not implemented.",
   "loc.messages.ExpiredServicePrincipal": "Could not fetch access token for Azure. Verify if the Service Principal used is valid and not expired. For more information refer https://aka.ms/azureappservicedeploytsg",
   "loc.messages.CorrelationIdForARM": "Correlation ID from ARM api call response : %s",
-  "loc.messages.CantDownloadAccessProfile": "Cannot download access profile/kube config file for the cluster %s. Reason %s.",
```

Comment:
I did not understand changes to these files ? What it has to do with your code changes ?

Reply:
Not sure why these lines were deleted and the others were added. I did not manually delete or add any of these lines, they were added when I built the project and connected it to docker-common-v2. I can revert these changes if need be.